lib/jxl/image_bundle.h

```diff
@@ -213,8 +213,15 @@ class ImageBundle {
   YCbCrChromaSubsampling chroma_subsampling;
 
   FrameOrigin origin{0, 0};
-  // Animation-related information. This assumes GIF- and APNG- like animation.
+
+  // Animation-related information, corresponding to the timecode and duration
+  // fields of the jxl::AnimationFrame of the jxl::FrameHeader.
+  // TODO(lode): ImageBundle is used here to carry the information from
+  // jxl::FrameHeader, consider instead passing a jxl::FrameHeader directly to
+  // EncodeFrame or having a field of that type here.
```

Comment:
+1 on having a FrameHeader field here. The current approach is incomplete: crop dimensions are given by whatever the frame dimensions happen to be, position is given by `origin`, `duration` is here, and blending is only specified in a simplified way (extra channel blending is ignored here, as well as clamping and the possibility to use different sources, etc).
It would be cleaner to replace the current fields by a single FrameHeader field, which would immediately make it complete.

Reply:
(obviously not for this PR)

Reply:
Acknowledged